README.rst

```diff
@@ -191,4 +195,43 @@ and similar for other locations (like 4240). Figure 6::
 
     python plot_effsel_location.py 4240 ./effsel_4240.ps 
 
-also similar for other locations.
\ No newline at end of file
+also similar for other locations.
+
+
+The age-metallicity structure of the Milky Way disk
+-----------------------------------------------------------------
+
+The following section concerns the adaptations made to the code above for the Mackereth et al. (2017) paper. Most of the additions were made in order to allow the use of the code with the full APOGEE red giant branch (RGB) sample, which has a larger sample with measured ages, and for which the selection function is more easily applied to the stellar evolution models (for calculating the surface-mass density contributions of populations).
+
+There are quite a few extra requirements necessary to reproduce the results which make up this paper, the main ones being the extra data tables (with DR12 RGB distances, and the ages from Martig et al. 2016), and the PARSEC isochrones - implemented via the `isodist <https://github.com/jobovy/isodist>`__ python package.
+
+To get hold of the DR12 distances, which have not yet been made publicly available, you need to have access to the SDSS SAS (SDSS collaborators only) and the file at `this link <https://data.sdss.org/sas/apogeework/apogee/sandbox/Distance_VAC/dr12/DR12_DIST_R-GC.fits>`__. You should place this file in ../catalogues . 
```

Comment:
Maybe we should say here that until these are made public, the analysis can be done using the DR14 value-added catalog on distances (+link); that would obviously require some tweaks, but we can just state that.